.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "ci"]
+	path = ci
+	url = https://github.com/fromanirh/traviskube
```

Comment:
What happens here if we move to kubevirt offical repo ?

Reply:
for this specific hunk I suggest to keep it like this to point to my repo for the time being. I have few changes that I'm not sure they can make quickly into u/s - and I surely need to polish them a bit.
After that we will have to change the git submodule ref.